src/polyhedralGravity/calculation/GravityModel.h

```diff
@@ -21,17 +21,6 @@
 
 namespace polyhedralGravity {
 
-    /**
```

Comment:
where did these go? :o

Reply:
In `model/GravityModelData.h`

Seems more fitting to have simple aliases there in order to reuse them in `SanityCheck.h`.